tests/Oryx.BuildImage.Tests/Python/PythonSampleAppsTest.cs

```diff
@@ -1124,8 +1124,8 @@ public void Django_CollectStaticFailure_DoesNotFailBuild()
         }
 
         [Theory]
-        [InlineData("3")]
-        [InlineData("2")]
+        [InlineData("3.8.5")]
+        [InlineData("2.7.18")]
         public void Build_ExecutesPreAndPostBuildScripts_WithinBenvContext(string version)
```

Comment:
this change is only to get rid of test failures in master .. not related to python packaging

Reply:
Just FYI...I merged my PR...you can now rebase on it.